src/Dock.Model.INPC/Controls/ToolDock.cs

```diff
@@ -8,5 +8,6 @@ namespace Dock.Model.Controls
     /// </summary>
     public class ToolDock : DockBase, IToolDock
     {
+        public IView FocusedView { get; set; }
```

Comment:
FocusedView should only be defined in DockBase and in the same way as this https://github.com/wieslawsoltes/Dock/blob/a86ba53de05652d1d9418c435b5e0982896ed657/src/Dock.Model.INPC/DockBase.cs#L51-L57